author	Jefferson Delfes <jefferson.delfes@openbossa.org>
commit	1801a4d45d74c29cebacaab79a07e9961300be41
	Sat, 2 Jun 2012 01:12:36 +0000 (09:12 +0800)

mgmtops: Fix endianness conversion

Flags field has 32-bit size.

diff -cr bluez-4.99.pan.orig/plugins/mgmtops.c bluez-4.99.pan/plugins/mgmtops.c
*** bluez-4.99.pan.orig/plugins/mgmtops.c	Mon Oct 15 05:42:40 2012
--- bluez-4.99.pan/plugins/mgmtops.c	Mon Oct 15 09:47:29 2012
***************
*** 1576,1582 ****
  	else
  		eir = ev->eir;
  
! 	flags = btohs(ev->flags);
  
  	ba2str(&ev->addr.bdaddr, addr);
  	DBG("hci%u addr %s, rssi %d flags 0x%04x eir_len %u",
--- 1576,1582 ----
  	else
  		eir = ev->eir;
  
! 	flags = btohl(ev->flags);
  
  	ba2str(&ev->addr.bdaddr, addr);
  	DBG("hci%u addr %s, rssi %d flags 0x%04x eir_len %u",
